src/SearchResults/index.js

```diff
@@ -714,9 +714,20 @@ function extractNormalizedFacetValues(results, attribute) {
 
     var hierarchicalFacet =
       results._state.getHierarchicalFacetByName(attribute);
-    var currentRefinementSplit = unescapeFacetValue(
+    var separator =
+      results._state._getHierarchicalFacetSeparator(hierarchicalFacet);
+    var currentRefinement = unescapeFacetValue(
       results._state.getHierarchicalRefinement(attribute)[0] || ''
-    ).split(results._state._getHierarchicalFacetSeparator(hierarchicalFacet));
+    );
+
+    if (currentRefinement.startsWith(hierarchicalFacet.rootPath)) {
```

Comment:
I think we need to keep to keep it ES5 compliant.

```suggestion
    if (currentRefinement.indexOf(hierarchicalFacet.rootPath) === 0) {
```

Reply:
Could we instead maybe do the replace without condition? I think it's being compared twice now already. 

Not sure if it's more efficient, but we could also make the first argument of the replace with a regex `new RegExp('^' + ....`. Although now I'm thinking of it, that may be slower

Reply:
Yeah I opted for Dhaya's suggestion, it's also clearer for sure